synapse/push/pusherpool.py

```diff
@@ -62,7 +62,7 @@ def __init__(self, hs: "HomeServer"):
         self.clock = self.hs.get_clock()
 
         # We shard the handling of push notifications by user ID.
-        self._pusher_shard_config = hs.config.push.pusher_shard_config
+        self._pusher_shard_config = hs.config.worker.pusher_shard_config
```

Comment:
I'm surprised this one used to work since it was specific, but pointing to the *wrong* one?